src/app/themes/theme.js

```diff
@@ -139,6 +139,9 @@ const theme = createTheme({
       containedSizeLarge: {
         fontSize: '1.125em',
       },
+      outlined: {
+        border: '#000 solid 1px',
```

Comment:
This is a global change to all buttons, right? Is it necessary? What does it change? How many places? I can't tell if it's being used anywhere. Would it be used on a "variant=outline" button? I don't see anything like that in this PR, and don't think I've used it before